spec/use_packwerk/private/interactive_cli_spec.rb

```diff
@@ -91,5 +102,60 @@ module UsePackwerk
       )
       subject
     end
+
+    it 'allows visualizing packs interactively' do
+      write_package_yml('packs/my_destination_pack', owner: 'Artists')
+      write_file('config/teams/artists.yml', 'name: Artists')
+
+      prompt.input << "Visualize\r" # Hello! What would you like to do?
+
+      prompt.input << INPUTS::DOWN # Do you want the graph nodes to be teams or packs?
+      prompt.input << INPUTS::RETURN # (Confirms 'Packs')
+
+      prompt.input << INPUTS::DOWN # Do you select packs by name or by owner?
+      prompt.input << INPUTS::RETURN # (Confirms 'owner')
+
+      prompt.input << 'Artists' # Please select team owners
+      prompt.input << INPUTS::SPACE # (Select it)
+      prompt.input << INPUTS::RETURN # (Confirm selection)
+      prompt.input << INPUTS::EOF
+
+      prompt.input.rewind
+
+      expect(VisualizePackwerk).to receive(:package_graph!).with([ParsePackwerk.all.first])
+
+      subject
+    end
+
+    it 'fails to visualize if no packs are selected' do
+      write_package_yml('packs/my_destination_pack', owner: 'Artists')
+      write_file('config/teams/artists.yml', 'name: Artists')
+
+      prompt.input << "Visualize\r" # Hello! What would you like to do?
+
+      prompt.input << INPUTS::DOWN # Do you want the graph nodes to be teams or packs?
```

Comment:
The comments definitely make it a lot nicer! It would be really nice if we could do something like `expect(prompt.output).to eq("Do you want the graph nodes to be teams or packs?")`, but I think as long as our subject calls so many prompts. I wonder if it's worth filing an issue in `tty-prompt` to ask how he suggests testing something like this, but probably not worth it.